resources/css/portal.css

```diff
@@ -492,5 +492,5 @@ select:not([class*="ant-"]) {
 }
 
 .ant-tooltip {
-  z-index: 999999;
+  z-index: 999999 !important;
```

Comment:
I think this isn't good solution for this but couldn't figure any other workaround since AntD styles overwrite this later on with `z-index: 50001;` or something similar.